celery_worker/worker.py

```
@@ -3,170 +3,183 @@
+                    "question": question,
+                    "thread_id": drawing_id,
+                    "page": page
+                }, timeout=300)
+                resp.raise_for_status()
+
+                result = resp.json()
+                if result.get("success"):
+                    answer = result.get("answer")
+                else:
+                    error_msg = result.get("error", "Агент вернул ошибку без описания")
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)